### ISSUE TITLE
improve startup speed (8 times faster)

### ISSUE THREAD

**Issue by Max-RM** [HiveGamesOSS/Chunker#7]
### What feature would you like seen in Chunker?

There is an exe for Windows in releases page. 
But if you unzip this exe as an archive and run another exe that is inside,
the launch will happen 8 times faster:

https://github.com/user-attachments/assets/318942e2-8b6d-46d8-a9e9-ed5ae14a3e08

So I suggest also publishing the unpacked exe in releases.
(Also in releases it called as x64 exe while task manager says that this is x86)

**Comment by SuperLlama88888** [comment]
speeed

**Comment by bytewired9** [comment]
yeah I'd much rater a chunker installer instead of a portable application (coming from a guy who loves portable applications)

**Comment by FormallyMyles** [comment]
> yeah I'd much rater a chunker installer instead of a portable application (coming from a guy who loves portable applications)

There are plans to hopefully have an installer version but unfortunately there isn't an exact timeframe on it.

**Comment by FormallyMyles** [comment]
We have made a new .zip version available in the latest release (which is signed) - https://github.com/HiveGamesOSS/Chunker/releases
